### DISCUSSION TITLE
I cant use copilot

### DISCUSSION THREAD

**Discussion by Penguin-SAMA** [community/community#78004] [GitHub Education]
### Hi everyone,

I already have the GitHub Student Pack benefits, but suddenly I no longer have access to Copilot.
![image](https://github.com/community/community/assets/50973632/e64dc681-3bcf-4709-bf98-a2f76798d8b4)
![image](https://github.com/community/community/assets/50973632/4d5ec914-3491-4557-a383-2444af6a56a1)

**Comment by IgnacioPalma** [comment]
Hello @Penguin-SAMA! :wave: 

Have you been able to get access to GitHub Copilot? :thinking:

**Comment by Penguin-SAMA** [comment]
@IgnacioPalma I cant

**Comment by IgnacioPalma** [comment]
@Penguin-SAMA I don't have the knowledge on what may be happening, but have you tried the steps on the following guide?: [link](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279)

**Comment by Penguin-SAMA** [comment]
@IgnacioPalma  I've tried it, but when I use Copilot, it shows that payment is required.
![image](https://github.com/community/community/assets/50973632/a75ce028-bc2f-4bc4-a559-c85ac8afbfe2)
and when I open https://education.github.com/discount_requests/application, it automatically redirects to https://github.com/pricing.
